app/views/tasks/_form.html.erb

```diff
@@ -4,7 +4,7 @@
   <div class="form-inputs">
     <%= f.input :title %>
     <%= f.input :priority, as: :select, collection: Task::PRIORITIES %>
-    <%= f.input :status, :as => :select, :collection => ['Active', 'Cancelled'] %>
+    <%= f.input :status, :as => :select, :collection => ['Active', 'Cancelled'], :selected => 'Active' %>
```

Comment:
This was much closer to what I was expecting, but it would be better to have this in the controller.

Reply:
Let me try that then

Reply:
Thanks !